### PR TITLE
update to remove github.com/pkg/errors

### PR DESCRIPTION
**Pull request by learner0810**
#### What type of PR is this?
/kind cleanup

#### Which issue(s) this PR fixes:

xref #103043

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs
NONE
```

### CODE REVIEW

File: test/e2e/storage/framework/volume_resource.go [hunk 1]
```diff
@@ -184,27 +183,27 @@ func (r *VolumeResource) CleanupResource() error {
...
+					cleanUpErrs = append(cleanUpErrs, fmt.Errorf( "failed to find PVC %v: %w", r.Pvc.Name,err))
 				}
 
 				if pv != nil {
 					err = e2epv.WaitForPersistentVolumeDeleted(f.ClientSet, pv.Name, 5*time.Second, 5*time.Minute)
 					if err != nil {
-						cleanUpErrs = append(cleanUpErrs, errors.Wrapf(err,
-							"Persistent Volume %v not deleted by dynamic provisioner", pv.Name))
+						cleanUpErrs = append(cleanUpErrs, fmt.Errorf(
+							"persistent Volume %v :%w not deleted by dynamic provisioner", pv.Name,err))
```
Comment (wzshiming) [inline test/e2e/storage/framework/volume_resource.go:206]:
```suggestion
							"persistent Volume %v : not deleted by dynamic provisioner: %w", pv.Name,err))
```

File: test/e2e/storage/external/external.go [hunk 2]
```diff
@@ -347,7 +346,7 @@ func loadSnapshotClass(filename string) (*unstructured.Unstructured, error) {
 	snapshotClass := &unstructured.Unstructured{}
 
 	if err := runtime.DecodeInto(scheme.Codecs.UniversalDecoder(), data, snapshotClass); err != nil {
-		return nil, errors.Wrap(err, filename)
+		return nil, fmt.Errorf("%s:%w",filename,err)
```
Comment (wzshiming) [inline test/e2e/storage/external/external.go:349]:
```suggestion
		return nil, fmt.Errorf("%s: %w",filename,err)
```

File: test/e2e/storage/external/external.go [hunk 3]
```diff
@@ -201,7 +200,7 @@ func loadDriverDefinition(filename string) (*driverDefinition, error) {
 	// TODO: strict checking of the file content once https://github.com/kubernetes/kubernetes/pull/71589
 	// or something similar is merged.
 	if err := runtime.DecodeInto(scheme.Codecs.UniversalDecoder(), data, driver); err != nil {
-		return nil, errors.Wrap(err, filename)
+		return nil, fmt.Errorf("%s:%w",filename,err)
```
Comment (wzshiming) [inline test/e2e/storage/external/external.go:203]:
```suggestion
		return nil, fmt.Errorf("%s: %w",filename,err)
```

**Comment by wzshiming** [comment]
Please execute `./hack/update-gofmt.sh` to format the code

**Comment by learner0810** [comment]
@wzshiming It's done

File: test/e2e/storage/framework/volume_resource.go [hunk 4]
```diff
@@ -184,27 +183,27 @@ func (r *VolumeResource) CleanupResource() error {
...
 					default:
-						cleanUpErrs = append(cleanUpErrs, errors.Wrapf(err, "Failed to find PVC %v", r.Pvc.Name))
+						cleanUpErrs = append(cleanUpErrs, fmt.Errorf("failed to find PVC %v: %w", r.Pvc.Name, err))
 					}
 				}
 
 				err := e2epv.DeletePersistentVolumeClaim(f.ClientSet, r.Pvc.Name, f.Namespace.Name)
 				if err != nil {
-					cleanUpErrs = append(cleanUpErrs, errors.Wrapf(err, "Failed to delete PVC %v", r.Pvc.Name))
+					cleanUpErrs = append(cleanUpErrs, fmt.Errorf("failed to find PVC %v: %w", r.Pvc.Name, err))
```
Comment (wzshiming) [inline test/e2e/storage/framework/volume_resource.go:199]:
```suggestion
					cleanUpErrs = append(cleanUpErrs, fmt.Errorf("failed to delete PVC %v: %w", r.Pvc.Name, err))
```

File: test/e2e/storage/framework/volume_resource.go [hunk 5]
```diff
@@ -184,27 +183,27 @@ func (r *VolumeResource) CleanupResource() error {
...
 				}
 
 				if pv != nil {
 					err = e2epv.WaitForPersistentVolumeDeleted(f.ClientSet, pv.Name, 5*time.Second, 5*time.Minute)
 					if err != nil {
-						cleanUpErrs = append(cleanUpErrs, errors.Wrapf(err,
-							"Persistent Volume %v not deleted by dynamic provisioner", pv.Name))
+						cleanUpErrs = append(cleanUpErrs, fmt.Errorf(
+							"persistent Volume %v: %w not deleted by dynamic provisioner", pv.Name, err))
```
Comment (wzshiming) [inline test/e2e/storage/framework/volume_resource.go:206]:
```suggestion
						cleanUpErrs = append(cleanUpErrs, fmt.Errorf(
							"persistent Volume %v not deleted by dynamic provisioner: %w", pv.Name, err))
```

File: test/e2e/storage/utils/create.go [hunk 6]
```diff
@@ -173,13 +172,13 @@ func CreateItems(f *framework.Framework, ns *v1.Namespace, items ...interface{})
 			if err == nil {
 				done = true
 				break
-			} else if errors.Cause(err) != errorItemNotSupported {
+			} else if errors.Unwrap(err) != errorItemNotSupported {
```
Comment (wzshiming) [inline test/e2e/storage/utils/create.go:175]:
```suggestion
			} else if !errors.Is(err, errorItemNotSupported) {
```

**Comment by learner0810** [comment]
@wzshiming sorry, I miss that ,done

**Pull request merged**